README.md

```diff
@@ -8,6 +8,39 @@ effectively together with [nginx's autoindex module](http://nginx.org/en/docs/ht
 
 A sample nginx configuration is also included which mounts **file browser** under root (`/`) and mounts files to be listed under `/files` path. Hence is the `filesBaseUrl` under
 
+## Installation / Configuration
+
+The system has a default to use the location's root (E.G. the site's root location `www.example.net/`, and the physical of `/opt/www/`). This can be configured to move both the ***installation*** location and the location to be browseable.
+
+1. Clone the repo to the server
+2. Create an installation location (E.G. `/var/www/files/`)
+3. Copy the `css`, `image`, `js` directories to the installation location
+4. Copy the `index.html` to the installation location
+5. Create the location for the files to be shared & browsable (E.G. `/var/www/isos`)
+6. Copy the files to the location
```

Comment:
technically seen, you don't need to copy. for many use cases copying (probably gigabytes of data) is not an option. so either copy or using a symlink (which is described a at the end of this doc).